service/src/main/java/bio/terra/catalog/service/dataset/DatasetDao.java

```diff
@@ -76,7 +79,7 @@ public Dataset upsert(Dataset dataset) {
   }
 
   @WriteTransaction
-  public Dataset update(Dataset dataset) {
```

Comment:
Why this change? It seems unrelated to me?

Reply:
It is unrelated. It was changed due to a sonar / IntelliJ warning because the method returns a value but the return value is never used. If at some point the return value is used, it can be added back.